deploy.go

```diff
@@ -304,6 +304,63 @@ func (gc *GatewayClient) ValidateMDMDetails(mdmTopologyParam []byte) (*types.Gat
 	return &gatewayResponse, nil
 }
 
+// GetClusterDetails used for get cluster details
```

Comment:
the comment can be improved as get MDM cluster details

Reply:
Updated.